_inc/client/components/welcome-new-plan/premium.jsx

```diff
@@ -13,6 +14,21 @@ import JetpackDialogue from 'components/jetpack-dialogue';
 import { imagePath } from 'constants';
 
 class WelcomePremium extends Component {
+	componentDidMount() {
+		analytics.tracks.recordEvent( 'jetpack_warm_welcome_plan_view', {
+			planClass: this.props.planClass,
+		} );
+	}
+
+	clickCtaDismiss( cta ) {
+		analytics.tracks.recordEvent( 'jetpack_warm_welcome_plan_click_' + cta, {
```

Comment:
It might make more sense to just have a singular `jetpack_warm_welcome_plan_click` event and pass the CTA as a prop (which it looks like we have already), then we can make simple funnels and segment by CTA.